### PR TITLE
fix(demo): fix indent between progressbar and Randomize buttons

### PR DESCRIPTION
**Pull request by ghost**
- added wrapper for progressbar blocks to set indent between progressbar block and `Randomize` buttons. http://joxi.ru/L216zjet69qYOA 
- deleted 'br' tags for consistency(replaced by wrappers)

Related to #3735

### CODE REVIEW

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
From where you've deleted br tags? How can I test this change (how it affects demo)?

**Comment by ghost** [comment]
change description in pr

**Comment by YevheniiaMazur** [comment]
Tested, looks good

File: demo/src/app/components/+progressbar/demos/dynamic/dynamic.html [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-
-<small><em>No animation</em></small>
-<progressbar [animate]="false" [value]="dynamic" type="success"><b>{{dynamic}}%</b></progressbar>
-
-<small><em>Object (changes type based on value)</em></small>
-<progressbar class="progress-striped active" [value]="dynamic" [type]="type">{{type}}</progressbar>
-
+<div class="mb-2">
+  <progressbar [max]="max" [value]="dynamic">
+    <span style="color:white; white-space:nowrap;">{{dynamic}} / {{max}}</span>
```
Comment (valorkin) [inline demo/src/app/components/+progressbar/demos/dynamic/dynamic.html]: @svetoldo4444ka please avoid inline styles

**Review by valorkin — CHANGES_REQUESTED** [review]
please extract inline styles

**Review by valorkin — APPROVED** [review]

**Pull request merged**